Src/WitsmlExplorer.Api/Query/ObjectQueries.cs

```diff
@@ -53,16 +52,18 @@ public static IWitsmlObjectList GetWitsmlObjectsByType(EntityType type)
         public static IWitsmlObjectList GetWitsmlObjectsWithParamByType(EntityType type, string objectProperty, string objectPropertyValue)
         {
             WitsmlObjectOnWellbore o = EntityTypeHelper.ToObjectOnWellbore(type);
-            o.UidWell = "";
-            o.UidWellbore = "";
-            o.Uid = "";
-            o.NameWell = "";
-            o.NameWellbore = "";
-            o.Name = "";
+            o.UidWell = string.Empty;
+            o.UidWellbore = string.Empty;
+            o.Uid = string.Empty;
+            o.NameWell = string.Empty;
+            o.NameWellbore = string.Empty;
+            o.Name = string.Empty;
             if (objectProperty != null)
             {
                 o = QueryHelper.AddPropertyToObject(o, objectProperty, objectPropertyValue);
             };
+
+            // TODO: REMOVE CASTING!
```

Comment:
Will this be remove in another task?

Reply:
Yes, it must be and will be removed in future, it is note that this casting is not save and should not be used in this way